Automattic/wp-calypso
# Untangling: apply new design to Settings -> Administration -> Transfer site

Fixes https://github.com/Automattic/dotcom-forge/issues/9860

## Proposed Changes

This PR applies the new global view design to Settings -> Administration -> Transfer site flow.

## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Open /sites.
2. Click a site.
3. Go to Settings -> Administration.
4. Click the `Transfer site` button.
5. Verify you see this:

<img width="995" alt="image" src="https://github.com/user-attachments/assets/34e0adac-89f6-47f7-8689-82cd437903b1">

6. Enter another username of yours and click `Continue`. Verify you see:

<img width="993" alt="image" src="https://github.com/user-attachments/assets/36332fbe-086d-4127-8428-ac6705708f48">

7. Tick all toggles and click `Start transfer`. Verify you see:

<img width="993" alt="image" src="https://github.com/user-attachments/assets/3600eebf-2ba3-4c1f-ae9f-1b44ed3c1e12">

8. Check your email with the subject `Confirmation needed to finish blog transfer`. Click the button:

<img width="432" alt="image" src="https://github.com/user-attachments/assets/c55d8f8a-3574-446a-9f59-d87cc5d6f3ba">

9. You will get redirected to `https://wordpress.com/settings/start-site-transfer/:slug?site-transfer-confirm=code`.
10. Manually change the URL to `https://<calypso-live-url>/sites/settings/administration/:slug/transfer-site?site-transfer-confirm=code`. Verify you see:

<img width="998" alt="image" src="https://github.com/user-attachments/assets/dcd4ee0b-77e9-433b-8dc2-af76d8e0a667">

11. Go to the recipient's email. Accept the transfer. Verify that the is transferred successfully.
12. REGRESSION: Transfer another site via `/settings/general/:slug`; verify the flow can still be completed.

## Pre-merge Checklist
